samples/scala-protobuf-eventsourced-counter/src/test/scala/com/example/CounterServiceIntegrationSpec.scala

```diff
@@ -96,7 +96,7 @@ class CounterServiceIntegrationSpec
       counter.value shouldBe 15
 
       // verify message published to topic
-      val Message(decEvent, md): Message[Decreased] = eventsTopic.expectOneTyped
+      val Message(decEvent, md) = eventsTopic.expectOneTyped[Decreased]
```

Comment:
Is this changed due to Scala 3? If so, then it's a good plan to move all samples to Scala 3. Scala 3 compiler being more strict, we want to catch possible issues earlier.

Reply:
it is indeed required by scala 3.